config/config.example.yml

```diff
@@ -455,13 +455,17 @@ jobs:
 #use_pubsub_feeds: false
 
 ##
-## HMAC signing key used for CSRF tokens and pubsub
+## HMAC signing key used for CSRF tokens, cookies and pubsub
 ## subscriptions verification.
 ##
+## Note: This parameter is mandatory and should be a random string.
+## Such random string can be generated on linux with the following
+## command: `pwdgen 20 1`
+##
 ## Accepted values: a string
 ## Default: <none>
 ##
-#hmac_key:
+hmac_key: "CHANGE_ME!!"
```

Comment:
My only fear is that people will keep `"CHANGE_ME!!"`. That's why there is a special case for searxng for the same reason: https://github.com/searxng/searxng/blob/8e8d8dabe9b17c9db8db7432c6bc063d9ae980d1/searx/webapp.py#L135

Reply:
Didn't think about that, thanks! It's fixed now.